rag_apps/graphrag_knowledge_system/src/graph_store.py

```
@@ -0,0 +1,238 @@
+        self.G: nx.DiGraph = self._load()
+
+    # ── Persistence ───────────────────────────────────────────────────────────
+
+    def _load(self) -> nx.DiGraph:
+        if self.persist_path.exists():
+            try:
+                with open(self.persist_path, "r", encoding="utf-8") as f:
+                    data = json.load(f)
+                return nx.node_link_graph(data, directed=True, multigraph=False)
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)